### DISCUSSION TITLE
📢[중요] 깃허브 작업 기록 남기는 방법📢

### DISCUSSION THREAD

**Discussion by KIM-KWANHUN** [1995-s-Project/1995-s_GroupWare#24] [ANNOUNCEMENT]
### 작성자

- [x] 근희
- [ ] 경훈
- [X] 관훈
- [ ] 은미
- [ ] 재민

### 카테고리

전체 공지

### 공지내역

📢**깃허브 작업 기록 남기는 방법**📢

✅**STEP 1✅
회의 및 전체 공유할 사항 또는 도움이 필요하다면 Discussions 에 작성하기**

![디스커션1](https://github.com/user-attachments/assets/90bdcb96-2480-44b0-8ec6-18beaa4bf294)
**상단 Discussions 탭 클릭**

![디스커션2](https://github.com/user-attachments/assets/0bb58de9-72c9-43e3-9902-4fc6cbab78fe)
**가운데 New discussion 클릭** 

![디스커션3](https://github.com/user-attachments/assets/ce25e667-11f1-4a35-b9b4-75c2d312198d)
**템플릿 선택(Get started)**

 
![디스커션4](https://github.com/user-attachments/assets/a25856b7-0253-4bc7-adaa-392035e05335)
**Add a title 에 제목 작성, 작성자 및 카테고리 선택 후 공유할 내용 작성한 다음 Start discussion 클릭**

✅STEP 2✅
**모든 작업들은 Issues 탭에 작성해서 공유하기
🚨[중요]🚨 사소한 작업이라도 기본 이슈 템플릿 선택 후 작성 부탁드립니다**

![이슈1](https://github.com/user-attachments/assets/2b01a286-0ba3-47c0-a20a-4954855a5d7f)
**상단 Issues 탭 클릭**

![이슈2](https://github.com/user-attachments/assets/ed2b45ee-15bb-42bd-b5b8-46457df4db67)
**New issue 클릭**

![이슈3](https://github.com/user-attachments/assets/6386eba2-535b-426a-b0c3-693d27d558c7)
**템플릿 선택(Get started)**

![이슈4](https://github.com/user-attachments/assets/59c9cb5c-bda3-4e61-bd05-bf3d7f20c6aa)
**오른쪽 확인하면 템플릿 작성자(Assignees), 관련 라벨 선택(Labels), 어떤 프로젝트에 이슈를 공유할 것인지 선택(Projects)**

![이슈5](https://github.com/user-attachments/assets/525c452a-3025-44e7-92eb-38250100c410)
**전부 양식에 맞게 작성  후 하단 Submit new issue 클릭**

**✅STEP 3✅
이슈 작성 후 Projects 에서 진행상황에 따라 드래그 해놓기**

![프로젝트1](https://github.com/user-attachments/assets/57427d92-fd84-47f9-924e-25029151f5d0)
**상단 Projects 탭 클릭**

![프로젝트2](https://github.com/user-attachments/assets/f198ec0c-560f-40be-9b73-eb0fffe7deb6)
**해당 주차에 맞는 프로젝트 클릭**

![프로젝트3](https://github.com/user-attachments/assets/3aee00a8-f557-497a-b1bf-980445a0af16)
**이슈 확인 후 [확인된 이슈] 드래그, 그 중 본인이 처리할 이슈에 대해
[할 일 목록]에 드래그 하여 옮긴 후 진행상황에 따라 [진행중], [완료] 로 드래그 하기**

**Comment by rraaccoon** [comment]
# 👍
